lib/emerald/nodes/list_base.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'treetop'
+require_relative 'node'
+
+# Base rule for lists of images, metas, styles, and scripts
+class ListBase < Node
+  def to_html(context)
+    elements[4].elements.map do |e|
+      attrs = e.elements[0].elements.map do |j|
+        "#{j.elements[0].text_value}='#{j.elements[2].to_html(context)}'"
```

Comment:
I think we only do escaped double-quotes, so if you do `attr "value with ' single quote"`, this will result in broken html. We should probably use double quotes, and then I'm not sure if we need to re-escape quotes in the original string. What does the HTML spec say about escaping characters in attributes? is that a thing?